src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/MarkDuplicatesSparkIntegrationTest.java

```diff
@@ -158,4 +158,24 @@ public void testMarkDuplicatesSparkIntegrationTestLocal(
             }
         }
     }
+
```

Comment:
Do you have tests that take name-sorted input? Can you create a ticket to add more tests that use name-sorted input?

Reply:
in another PR